src/app/[locale]/layout.tsx

```
@@ -1,16 +1,18 @@
 import '@/styles/global.css';
-import { Providers } from "@/components/theme-provider"
 
+import { GeistMono } from 'geist/font/mono';
+import { GeistSans } from 'geist/font/sans';
 import type { Metadata } from 'next';
 import { NextIntlClientProvider, useMessages } from 'next-intl';
 import { unstable_setRequestLocale } from 'next-intl/server';
+
 import { GeistSans } from 'geist/font/sans';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'GeistSans'. Consider to delete it or rename it.

'GeistSans' is defined here:


(lint/suspicious/noRedeclare)